sdcm/results_analyze/__init__.py

```diff
@@ -571,7 +571,8 @@ def check_regression(self, test_id, is_gce=False, email_subject_postfix=None, us
         ycsb = doc['_source']['test_details'].get('ycsb')
         dashboard_path = "app/kibana#/dashboard/03414b70-0e89-11e9-a976-2fe0f5890cd0?_g=()"
 
-        last_events, events_summary = self.get_events()
+        last_events, events_summary = self.get_events(event_severity=[
+            Severity.CRITICAL.name, Severity.ERROR.name, Severity.DEBUG.name])
```

Comment:
isn't warning interesting to have in the emails too?

Reply:
Request was for this 3